### DISCUSSION TITLE
Why compose feel laggy

### DISCUSSION THREAD

**Discussion by hellosagar** [RajashekarRaju/hangman-compose#18] [General]
I tried the release version of the project also but the dialog popping up or red circle progressing it feels lagging? or something else?

**Comment by hellosagar** [comment]
I tested it on different devices and I found out that its just my device which feels laggy
sorry for false alarm

**Reply by RajashekarRaju** [reply]
Oh sorry for late reply, last week was chaos.
I'm glad it's working good fine.

Will soon work on new features.
